types/lru-cache/lru-cache-tests.ts

```diff
@@ -80,32 +80,32 @@ cache.prune();
 cache.forEach(function(value, key, cache) {
     value; // $ExpectType Foo
     key; // $ExpectType string
-    cache; // $ExpectType Cache<string, Foo>
-    this; // $ExpectType Cache<string, Foo>
+    cache; // $ExpectType LRUCache<string, Foo>
+    this; // $ExpectType LRUCache<string, Foo>
 });
 cache.forEach(function(value, key, cache) {
     value; // $ExpectType Foo
     key; // $ExpectType string
-    cache; // $ExpectType Cache<string, Foo>
+    cache; // $ExpectType LRUCache<string, Foo>
     this; // $ExpectType { foo(): void; }
 }, foo);
 
 cache.rforEach(function(value, key, cache) {
     value; // $ExpectType Foo
     key; // $ExpectType string
-    cache; // $ExpectType Cache<string, Foo>
-    this; // $ExpectType Cache<string, Foo>
+    cache; // $ExpectType LRUCache<string, Foo>
+    this; // $ExpectType LRUCache<string, Foo>
 });
 cache.rforEach(function(value, key, cache) {
     value; // $ExpectType Foo
     key; // $ExpectType string
-    cache; // $ExpectType Cache<string, Foo>
+    cache; // $ExpectType LRUCache<string, Foo>
     this; // $ExpectType { foo(): void; }
 }, foo);
 
 cache.keys(); // $ExpectType string[]
 cache.values(); // $ExpectType Foo[]
 
 const dump = cache.dump();
-dump; // $ExpectType LRUEntry<string, Foo>[]
+dump; // $ExpectType Entry<string, Foo>[]
```

Comment:
Command 

```
$ npm run lint lru-cache
```
returns the following error for this line:

```
/home/kostya/proj/DefinitelyTyped/types/lru-cache/lru-cache-tests.ts:110:1
ERROR: 110:1  expect                           Expected type to be:
  LRUCache.Entry<string, Foo>[]
got:
  Entry<string, Foo>[]

    at /home/kostya/proj/DefinitelyTyped/node_modules/dtslint/bin/index.js:182:19
    at Generator.next (<anonymous>)
    at fulfilled (/home/kostya/proj/DefinitelyTyped/node_modules/dtslint/bin/index.js:5:58)
    at <anonymous>
```

This is strange, because actual type is `LRUCache.Entry` and `Entry` interface is not available in this context. Maybe this is problem with `dtslint` :thinking: